src/scss/ucf-promo.scss

```diff
@@ -32,9 +33,9 @@ $screen-lg-min: 1200px;
 .promo-horizontal {
   padding: 30px;
 
-  h2 {
+  .promo-header {
     color: #fff;
-    font-family: 'Knockout 49 A','Knockout 49 B',Helvetica,Arial,sans-serif;
+    font-family: $font-family-condensed;
     font-size: 50px !important;
```

Comment:
To make the plugin more reusable, forcing !important overrides to make the main site stylesheet behave would be better handled at the main site level.  Not a huge deal right now, since we're building this primarily for use on the main site, but maybe consider adding a backlog task for it if you're short on time.